backend/src/dao/MembersDao.ts

```diff
@@ -96,6 +96,11 @@ export default class MembersDao extends BaseDao<IdolMember, IdolMember> {
     await batch.commit();
   }
 
+  /**
+   * Gets all teams based on the subteam and formersubteam(s) each member is on, since teams do not have
+   * a collection in the database (teams are aggregated data from the members collection).
+   * @returns A promise that resolves to an array of all Team objects.
```

Comment:
Saying "all Team objects" could be a little confusing here because it implies theres a set of team objects somewhere out there when we're really just constructing these teams from scratch.